packages/changefeed/src/tests/mongo.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable functional/no-loop-statements, functional/no-let, functional/immutable-data, functional/prefer-immutable-types, promise/param-names, no-restricted-syntax, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any, max-lines-per-function */
```

Comment:
@coderabbitai write an issue to deal with this eslint ignore header. That is a lot of stuff ignored.